Assets/MixedRealityToolkit.Staging/UnityAR/Editor/ConfigurationChecker/UnityARConfigurationChecker.cs

```diff
@@ -11,82 +13,132 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UnityAR
     /// Class to perform checks for configuration checks for the UnityAR provider.
     /// </summary>
     [InitializeOnLoad]
-    public class UnityARConfigurationChecker
+    static class UnityARConfigurationChecker
```

Comment:
Should we explicitly add `private` here?

Reply:
the language does not allow namespace members to be marked as private

![image](https://user-images.githubusercontent.com/13281406/71755753-1f3c0300-2e41-11ea-8002-fb10fc224047.png)